content/applications/websites/livechat/overview/responses.rst

```diff
@@ -1,48 +1,137 @@
+      :guilabel:`Livechat`.
+
+      .. image:: responses/live-chat-responses-lead.png
+         :align: center
+         :alt: View of the results from a /lead command in a Live Chat conversation.
+
+   .. tab:: `/leave`
+
+      The `/leave` command lets the *operator* leave the conversation channel.
 
 .. important::
-   | - For *helpdesk tickets*: make sure the application is installed on your database and the
-     option *Live Chat* under :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`
-     is enabled.
-   | - For *leads*: the *CRM* application must be installed on your database.
+   - The `/helpdesk` and `/helpdesk_search` commands can only be used if the :guilabel:`Helpdesk`
+     app has been installed, and :guilabel:`Live Chat` has been activated on a :guilabel:`Helpdesk
+     Team`. To activate :guilabel:`Live Chat`, go to :menuselection:`Helpdesk --> Configuration -->
+     Teams`, and select a team. Scroll to the :guilabel:`Channels` section and check the box
+     labeled :guilabel:`Live Chat`.
+   - The `/lead` command can only be used if the :guilabel:`CRM` app has been installed.
+
+.. tip::
+   To access the :guilabel:`ticket` or :guilabel:`lead` created from the chat, click on the link.
 
-To access the ticket or lead created from the chat, click on the shortcut link.
+  .. image:: responses/live-chat-responses-ticket-link.png
+     :align: center
+     :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat.
 
-.. image:: responses/create_ticket.png
-   :align: center
-   :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat
+.. seealso::
+   - :doc:`/applications/sales/crm/acquire_leads`
+   - :doc:`/applications/services/helpdesk/overview/getting_started`
+
+Canned responses
+================
+
+*Canned responses* are customizable inputs where a shortcut stands in for a longer response. An
+operator will enter the :guilabel:`Shortcut`, and it will be replaced by the
+:guilabel:`Substitution`.
+
+Create new canned responses
+---------------------------
+
+To create a new :guilabel:`Canned Response`, go to :menuselection:`Live Chat --> Configuration -->
+Canned Responses --> New`. Choose a :guilabel:`Shortcut`. Enter the :guilabel:`Substitution`
+message. This will be the message that is sent to the recipients. Click :guilabel:`Save`.
 
 .. tip::
-   Helpdesk tickets created from the chat automatically add the conversation as a description of
-   the ticket. The same goes for the creation of a lead.
+  Try to connect the :guilabel:`Shortcut` to the topic of the :guilabel:`Substitution`. The easier
+  it is for the *operators* to remember, the easier it will be to use the *canned responses* in
+  conversations.
+
+Insert a canned response in a conversation
+------------------------------------------
 
-Send canned responses
-=====================
+To use a *Canned response* during a live chat conversation, type a colon (`:`)  into the chat
```

Comment:
Question: Should "Live Chat" have a :guilabel: here? Not quite sure - could be worth checking in with ZST about this.

Also, however you decide to format Canned Responses (i.e. Canned responses, canned responses, :guilabel:`Canned Responses`, *Canned responses*, or *Canned Responses*) just make sure you stick with one all the way through the doc to ensure uniformity throughout.

Reply:
..., start by typing a colon...

Reply:
Since I am referring to  a "live chat conversation", and not specifically the application name, I used lower case and no guilabel, as is the industry standard (it is also how we currently refer to live chat on our website, and when referring to our own internal live chat conversations).